proto/buf/alpha/registry/v1alpha1/plugin.proto

```diff
@@ -48,6 +48,8 @@ message PluginVersion {
   string plugin_name = 3;
   // The owner of the plugin to which this version relates.
   string plugin_owner = 4;
+  // The full docker image digest of this plugin version.
+  string image_digest = 5;
```

Comment:
`image` has a specific meaning in Buf, this is more confusing for me - I think it's worth talking about with the team